src/app/features/suggested-first-steps/components/add-funds-step.tsx

```diff
@@ -14,8 +14,8 @@ interface AddFundsStepProps {
 export function AddFundsStep({ isComplete, onSelectStep }: AddFundsStepProps) {
   return (
     <SuggestedFirstStep
-      action="Get STX"
-      body="Get some STX so you can start using apps"
+      action="Add funds"
+      body="Get some STX for using with apps"
```

Comment:
or BTC?

Reply:
Good suggestion, added BTC first given this is a BTC first wallet.

Reply:
Happy to make this change now here though note we won't have BTC support on the fund page (aside from the receive modal?) until we tackle https://github.com/hirosystems/stacks-wallet-web/issues/3188